CONTRIBUTING.md

```diff
@@ -1,5 +1,13 @@
-Contributing
-------------
+# Contributing
+--------------
```

Comment:
No need for the underline here since you are using `#` to mark "Contributing" as heading.
Markdown renders the `-------` as an `<hr>` which looks odd imho

Reply:
Well, I agree,  but that line was already there :-)

Reply:
The line was there because Markdown accepts both `#` and the underlines to mark headings - https://daringfireball.net/projects/markdown/syntax#header  - so either leave the underline or replace it wholly with the hashtag :)